testutil/test_helper.go

```diff
@@ -2,17 +2,19 @@ package testutil
 
 import (
 	"bytes"
+	"encoding/json"
+	"errors"
 	"flag"
 	"fmt"
+	log "github.com/sirupsen/logrus"
```

Comment:
It's weird that your IDE is mixing github imports with the stdlib's... and apparently the linter isn't catching that :thinking: